nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/security/SecurityConstants.java

```diff
@@ -132,4 +132,10 @@
      */
     String MANAGE_LEGAL_HOLD = "ManageLegalHold";
 
+    /**
+     * Permission needed to move/transfer documents to cold storage.
+     *
```

Comment:
`Permission needed to move/transfer documents to cold storage.` -> can you please rename to include the fact that we transfer the content of the document instead of `move/transfer documents to cold storage.` -> the idea is to be consistent with what we did:
for example javadoc of the move method on the helper:
`Moves the main content associated with the document of the given {@link DocumentRef} to a cold storage...`

Reply:
`ManageLegalHold ` should be added in a xml contribution to be visible outer internal Nuxeo code,  can you have a look at [here](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/permissions-contrib.xml) I think without this contribution, the permission will not be visible on the web-ui ...to check

Reply:
I'll do it, thanks! 👍

Reply:
thanks, sorry i meant `WriteColdStorage` but i think you got it